files/docker-seccomp-default-with-personality.json

```diff
@@ -815,4 +840,4 @@
 			"excludes": {}
 		}
 	]
-}
+}
```

Comment:
Since it came up in an offline discussion. The newline change here is present in upstream which is why it's reflected in this diff.

 I actually cannot recall whether I initially created this file by copy-pasting and thus the final newline was added by my editor's (very appropriate) general policy of always having one or whether this is a change that occurred upstream as well but this file was fetched directly from the upstream tree via GitHub with no final newline.